src/audio_worklet.js

```diff
@@ -164,6 +145,7 @@ class BootstrapMessages extends AudioWorkletProcessor {
     // scope to create the real AudioWorkletProcessors that call out to Wasm to
     // do audio processing.
     let p = globalThis['messagePort'] = this.port;
+    /** @suppress {checkTypes} */
     p.onmessage = async (msg) => {
       let d = msg.data;
       if (d['_wpn']) {
```

Comment:
Now that this is part of the main JS, we can eventually go with expressive names that get minimised, right?

Reply:
I think so yes.